### ISSUE TITLE
Bootstrap 5 version?

### ISSUE THREAD

**Issue by toiyabe** [kaoz70/grapesjs-blocks-bootstrap4#94]
Hello!
I am wondering if there is a version of this project for Bootstrap 5? I found lots of what appears to be clones of this project, but none seem to be working.   This is a terrific project, and was hoping there is a version of this for 5.
Thanks!

**Comment by kaoz70** [comment]
Hi! There is no Bootstrap version 5 is available, I would love to have the time to upgrade it, but unfortunately I can't for now. I don't even know the scope of this, I stopped using Bootstrap a few years ago, so it may be an easy upgrade.

**Comment by toiyabe** [comment]
After searching around for a Bootstrap 5 version available, all the ones I could find were either broken (wouldn't work), deprecated, or no longer offered, or not maintained in a while.  Since this project works real well, and, I did some quick updates on classes, and it seems to work.  Here is a [link to the repo](https://github.com/MoonriseSoftwareCalifornia/grapesjs-blocks-bootstrap5). I'll try and keep that one in sync with any changes that may occur here. I'd appreciate any feedback on the new repo :)

**Comment by kaoz70** [comment]
Awesome, if you make any core changes it would be great to merge into this one repo as well, if they make sense. I have updated the readme with a link to your repo.

**Comment by toiyabe** [comment]
> Awesome, if you make any core changes it would be great to merge into this one repo as well, if they make sense. I have updated the readme with a link to your repo.

So far all the changes made were just with class differences based on [this excellent article](https://blog.abirana.com/how-to-migrate-convert-bootstra-4-to-bootstrap-5).  Also, I did make a change in the "Category" names so the user can distinguish between "Bootstrap" and "vanilla" GrapesJS items. I'm not sure the category name changes would go into this repo.

I have a screen shot of what I mean below.

![image](https://github.com/user-attachments/assets/6bdb394f-c9e9-4e44-bd23-930423a4624c)

In my project I'm adding this plugin for users of Bootstrap 4, the one mentioned above for Bootstrap 5, and another I found for Tailwind.  If a user references one of these three frameworks, [my app](https://github.com/MoonriseSoftwareCalifornia/CosmosCMS?tab=readme-ov-file#cosmos) is being built to automatically add the appropriate plugin.

@kaoz70, you mentioned you are not longer using BS.  What are you using now? Thanks for your work by the way!

**Comment by kaoz70** [comment]
Yeah, I would keep the category names out for now at least. I'm working with Tailwind on personal projects and at work we use EmotionJs

**Comment by toiyabe** [comment]
Is there a GrapesJS plugin for that by chance?

**Comment by kaoz70** [comment]
Well, looks like you got Tailwind covered but EmotionJs is not a util library like Tailwind, or a UI library like Bootstrap; it's more akin to SASS instead. So I don't think it will fit into GrapesJs.
